description.txt

```diff
@@ -1,10 +1,10 @@
-pysatMissionPlanning allows users to run build simulated satellites for TLE info and add empirical data. It includes the pysat_sgp4 instrument module which can be imported into pysat.
+pysatMissions allows users to run build simulated satellites for TLE info and add empirical data. It includes the pysat_sgp4 instrument module which can be imported into pysat.
 
 Main Features
 -------------
 - Simulate satellite orbits from TLEs and add data from empirical models
 - Import ionosphere and thermosphere values through pyglow
-- Import coordidnates through apexpy
+- Import coordinates through apexpy
```

Comment:
May want to list more than just apexpy here, or recast as more general, import coords from community packages